tests/acceptance/features/coreApiShareCreateSpecialToShares2/createShareWithDisabledUser.feature

```diff
@@ -23,6 +22,4 @@ Feature: share resources with a disabled user
     And user "Brian" has been created with default attributes and without skeleton files
     And user "Alice" has been disabled
     When user "Alice" shares file "textfile0.txt" with user "Brian" using the sharing API
-    Then the OCS status code should be "997"
-    #And the OCS status code should be "401"
```

Comment:
this steps are also in expected to failures. Updating line numbers is required.

Reply:
These steps are passed and thus removed from expected failure file